src/albibong/__init__.py

```diff
@@ -13,7 +14,7 @@
 from albibong.threads.websocket_server import get_ws_server
 
 logger = Logger(__name__, stdout=True, log_to_file=False)
-
+PORT = random.randrange(8500,8999)
```

Comment:
I got the idea but instead of a random function mutch better is to create a function that will find the first available port in some range. Also, it could be great to allow users to put custom ports in env variables. So I will remember to which process to kill before re run project